AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift

```diff
@@ -79,6 +80,11 @@ struct ChooseSessionTypeView: View {
                     didTapFixedSession = false
                 }
             }
+            .onAppear {
+                if tabSelection.selection == .createSession && tabSelection.mobileProceeding {
```

Comment:
related to my comment in `TabBarSelection`  - I have no idea what it accomplishes without looking at the Trello card. I feel like this is the easiest, but also the messiest way of modeling this. Consider changing it to something more robust and self explanatory